examples/no-prefix/main.tf

```diff
@@ -14,21 +14,38 @@ data "ibm_resource_group" "existing_resource_group" {
   name  = var.resource_group
 }
 
+#############################################################################
+# Locals
+#############################################################################
+locals {
+  access_tags                            = []
+  enable_vpc_flow_logs                   = true
+  cos_plan                               = "standard"
+  cos_location                           = "global"
+  create_authorization_policy_vpc_to_cos = true
+  network_cidrs                          = ["10.0.0.0/8", "164.0.0.0/8"]
+  address_prefixes = {
+    zone-1 = ["10.10.10.0/24"]
+    zone-2 = ["10.20.10.0/24"]
+    zone-3 = ["10.30.10.0/24"]
+  }
+}
+
 #############################################################################
 # Provision cloud object storage and bucket
 #############################################################################
 
 resource "ibm_resource_instance" "cos_instance" {
-  count             = var.enable_vpc_flow_logs ? 1 : 0
+  count             = local.enable_vpc_flow_logs ? 1 : 0
   name              = "${var.prefix}-vpc-logs-cos"
   resource_group_id = var.resource_group != null ? data.ibm_resource_group.existing_resource_group[0].id : ibm_resource_group.resource_group[0].id
   service           = "cloud-object-storage"
-  plan              = var.cos_plan
-  location          = var.cos_location
+  plan              = local.cos_plan
+  location          = local.cos_location
 }
 
 resource "ibm_cos_bucket" "cos_bucket" {
-  count                = var.enable_vpc_flow_logs ? 1 : 0
+  count                = local.enable_vpc_flow_logs ? 1 : 0
```

Comment:
no need for a count now that we are hard coding `enable_vpc_flow_logs` to `true`